src/System.Threading.Tasks/tests/System.Threading.Tasks.Tests.csproj

```diff
@@ -5,8 +5,14 @@
     <OutputType>Library</OutputType>
     <AssemblyName>System.Threading.Tasks.Tests</AssemblyName>
     <ProjectGuid>{B6C09633-D161-499A-8FE1-46B2D53A16E7}</ProjectGuid>
-    <NugetTargetMoniker>.NETStandard,Version=v1.5</NugetTargetMoniker>
+    <NugetTargetMoniker Condition="'$(TargetGroup)'=='netstandard1.7' AND '$(NugetTargetMoniker)'==''">.NETStandard,Version=v1.7</NugetTargetMoniker>
```

Comment:
I think you need to remove the second part of this condition as NugetTargetMoniker is always set when TargetGroup is netstandard1.7